packages/extension-ui/src/Popup/Forget.tsx

```diff
@@ -16,70 +17,164 @@ interface Props extends RouteComponentProps<{ address: string }>, ThemeProps {
   className?: string;
 }
 
-function Forget ({ className, match: { params: { address } } }: Props): React.ReactElement<Props> {
+function Forget({
+  className,
+  match: {
+    params: { address }
+  }
+}: Props): React.ReactElement<Props> {
   const { t } = useTranslation();
   const onAction = useContext(ActionContext);
   const [isBusy, setIsBusy] = useState(false);
 
-  const _goHome = useCallback(
-    () => onAction('/'),
-    [onAction]
-  );
+  const _goTo = useCallback((path: string) => () => onAction(path), [onAction]);
 
-  const _onClick = useCallback(
-    (): void => {
-      setIsBusy(true);
-      forgetAccount(address)
-        .then(() => {
-          setIsBusy(false);
-          onAction('/');
-        })
-        .catch((error: Error) => {
-          setIsBusy(false);
-          console.error(error);
-        });
-    },
-    [address, onAction]
-  );
+  const _onClick = useCallback((): void => {
+    setIsBusy(true);
+    forgetAccount(address)
+      .then(() => {
+        setIsBusy(false);
+        onAction('/');
+      })
+      .catch((error: Error) => {
+        setIsBusy(false);
+        console.error(error);
+      });
+  }, [address, onAction]);
+
+  const Helper = styled.div`
+  display: flex;
+  position: relative;
+  flex-direction: row;
+  justify-content: center;
+  align-items: flex-start;
+  gap: 12px;
+  padding-top: 0px;
+  margin-bottom: 16px;
+
+  &:before {
+  position: absolute;
+  content: '';
+  width: calc(100% - 32px);
+  border-top: 1px solid ${({ theme }: ThemeProps) => theme.boxBorderColor};
+  top: -16px;
+ }
+
+  img.icon {
+    margin-top: -4px;
+    align-self: flex-start; 
+  }
+
+  span {
+    font-weight: 300;
+    font-size: 13px;
+    line-height: 130%;
+    letter-spacing: 0.06em;
+    color: ${({ theme }: ThemeProps) => theme.subTextColor};
+    align-self: flex-start;
+
+    .link {
```

Comment:
Having Styled components on board, it's better to use them instead of global classes. E.g.:

```ts
const Link = styled.div`
  color: ${({ theme }: ThemeProps) => theme.primaryColor};
  cursor: pointer;
  :hover {
    text-decoration: underline;
   }
`
```